packages/select/src/choice.mts

```diff
@@ -0,0 +1,14 @@
+import { Separator } from '@inquirer/core';
+
+export type Choice<Value> = {
```

Comment:
I do appreciate trying to clean some code, but it'll be much easier for me to review a small diff.

I'm not against reviewing some refactors; but those should go into separate PRs. When adding a new feature, it helps a lot to keep the diff minimal and surgical to lower the review burden.

Reply:
I apologize for the cognitive overhead.